test/dhcp/conf_parser_test.rb

```diff
@@ -473,6 +473,11 @@ def test_include
                  included.select {|node| node.class == Proxy::DHCP::CommonISC::ConfigurationParser::IpV4SubnetNode}.map(&:subnet_address)
     assert_equal ['test.example.com'],
                  included.select {|node| node.class == Proxy::DHCP::CommonISC::ConfigurationParser::HostNode}.map(&:fqdn)
+    included = Proxy::DHCP::CommonISC::ConfigurationParser.new.subnets_hosts_and_leases(File.read('test/fixtures/dhcp/dhcp2.conf'), 'test/fixtures/dhcp/dhcp2.conf').flatten
+    assert_equal ['192.168.122.0', '192.168.123.0', '192.168.124.0', '192.168.1.0'],
+                 included.select {|node| node.class == Proxy::DHCP::CommonISC::ConfigurationParser::Ipv4Subnet}.map(&:subnet_address)
+    assert_equal ['test.example.com'],
+                 included.select {|node| node.class == Proxy::DHCP::CommonISC::ConfigurationParser::Host}.map(&:name)
   end
```

Comment:
Please create separate test for this named `test_include_relative`. Test two cases:

* include "dhcp_subnets.conf"
* include "../dhcp/dhcp_subnets.conf"

Only assert one thing in the file, for example `assert_equal ['test.example.com'], ...` would be a good candidate. Others are not relevant.